test/e2e/tests/test_table.py

```diff
@@ -104,22 +170,68 @@ def test_table_update_tags(self, forum_table):
         cr = k8s.wait_resource_consumed_by_controller(ref)
 
         # Update table list of tags
-        tags = [
+        new_tags = [
             {
                 "key": "key1",
                 "value": "value1",
             },
         ]
-        cr["spec"]["tags"] = tags
+        cr["spec"]["tags"] = new_tags
+
+        # Patch k8s resource
+        k8s.patch_custom_resource(ref, cr)
+        time.sleep(MODIFY_WAIT_AFTER_SECONDS * 2)
+
+        table_tags = get_resource_tags(cr["status"]["ackResourceMetadata"]["arn"])
+        tags.assert_ack_system_tags(
+            tags=table_tags,
+        )
+        tags_dict = tags.to_dict(
+            cr["spec"]["tags"],
```

Comment:
As an added layer of validation, consider using `new_tags` here instead of `cr["spec"]["tags"]`